extensions/tokenizers/rust/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "djl"
+version = "0.1.0"
+authors = ["Frank Liu <frankfliu2000@gmail.com>"]
+edition = "2021"
+
+[dependencies]
+jni = "0.19.0"
+tokenizers = { path = "../tokenizers/tokenizers", version = "*" }
+
+[lib]
+crate_type = ["cdylib"]
```

Comment:
Have you tried building a staticlib with JNI before? It would be easier to reuse.
(we gave it 1h, but didn't succeed on our side with JNA)

Reply:
This module is purely used for JNI, I not sure where else will reused the staticlib. We can make it staticlib if a use case comes up.